README.md

```diff
@@ -26,7 +26,7 @@ use bitcoincore_rpc::{Auth, Client, RpcApi};
 
 fn main() {
 
-    let rpc = Client::new("http://localhost:8332",
+    let rpc = Client::new("http://127.0.0.1:8332",
```

Comment:
I'm pretty skeptical of this change, TBH. It explicitly encodes ipv4 and will break if Core is not listening on ipv4.

Reply:
Understood, maybe just revert it back and add a short caveat at the bottom for those on macOS? its a pretty subtle "feature" of using this crate with macOS - kinda frustrating to run into (I had known about this actually but completely forgot).

Reply:
I'm curious if there's any other network client crate that gets this "right".

Definitely happy to add some extra text to the README suggesting that people try replacing `localhost` with `127.0.0.1`. But I think we should leave the default example code as is.